tests/acceptance/features/apiSpaces/search.feature

```diff
@@ -50,3 +50,51 @@ Feature: Search
       | /SubFolder1                                |
       | /SubFolder1/subFOLDER2                     |
       | /SubFolder1/subFOLDER2/insideTheFolder.txt |
+    And for user "Brian" the search result should contain space "mountpoint/folder"
```

Comment:
Brian seems to be searching for `folder` but this step says search result should contain `space`?
or may be `And for user "Brian" the search result should contain folder "folder" inside of space "Shares Jail"` 
 this type of step would fit in here? @ScharfViktor what do you think?

Reply:
Sorry that I didn't give you more context https://github.com/owncloud/ocis/pull/4463#issue-1351837237

Reply:
`And for user "Brian" the search result should contain folder "folder" inside of space "Shares Jail"` 
From the web's point of view, it looks the right thing to do, but `folder` is not realy folder, it's space (type mountpoint). So I search result gives me space. I would suggest to keep my version